spec/fake_braintree/address_spec.rb

```diff
@@ -1,14 +1,21 @@
 require 'spec_helper'
 
 describe "Braintree::Address.create" do
-  it "successfully creates address with valid data" do
-    customer_response = create_customer
+  let(:customer) { create_customer.customer }
 
+  it "successfully creates address with valid data" do
     address_response = Braintree::Address.create(
-      customer_id: customer_response.customer.id,
+      customer_id: customer.id,
       postal_code: 30339
     )
 
     expect(address_response).to be_success
   end
+
+  it "sets the creation time" do
+    address = Braintree::Address.create(customer_id: customer.id).address
+
+    creation_time = Time.parse(address.created_at)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.